OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/DbxAutoSync.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package latexstudio.editor;
+
+import javax.swing.SwingUtilities;
+import latexstudio.editor.remote.Cloud;
+import latexstudio.editor.remote.DbxUtil;
+
+/**
+ *
+ * @author White Hsu
+ */
+public class DbxAutoSync extends Thread {
```

Comment:
A preferred way of introducing anew thread is by implementing the Runnable interface (see PDFPreviewRefresher as example)